README.md

```diff
@@ -1,86 +1,99 @@
 # IAM-CTL
```

Comment:
What does CTL stand for? 
Couldn't find a generic explanation for it on the internet.

Reply:
Some  general comments:

- If this is specific to WSO2 IS, use `WSO2 IS or WSO2 Identity Server` instead of `Identity Server`. If we are referring to any identity server, use sentence case for `identity server`.
- In procedures, make sure the alignment is correct.
- Do not have two consecutive headings without any text in between.

Reply:
CTL should stand for Controller - as in `kubectl` for kubernetes cli tool